src/ast/mod.rs

```diff
@@ -4050,6 +4051,16 @@ pub enum Statement {
         arguments: Vec<Expr>,
         options: Vec<RaisErrorOption>,
     },
+    /// Return (SQL Server)
+    ///
+    /// for Functions:
+    /// RETURN
+    /// RETURN scalar_expression
+    /// RETURN select_statement
```

Comment:
1. Perhaps this example should be removed, until it's supported?
2. Do we want a "ReturnType" enum to describe what will be?

Reply:
Excellent. I will do this

Reply:
We'll want bare `RETURN` for SQL Server trigger bodies, so the followup on this is over here: https://github.com/apache/datafusion-sqlparser-rs/pull/1810